lldb/source/Plugins/Platform/gdb-server/PlatformRemoteGDBServer.cpp

```diff
@@ -684,6 +684,15 @@ Status PlatformRemoteGDBServer::RunShellCommand(
                                           signo_ptr, command_output, timeout);
 }
 
+bool PlatformRemoteGDBServer::CalculateMD5(const FileSpec &file_spec,
+                                           uint64_t &low, uint64_t &high) {
+  if (IsConnected()) {
```

Comment:
Suggestion: Flip the condition and do an early return with false.

Reply:
Should I change the other functions in `PlatformRemoteGDBServer` to do the same? For example `PlatformRemoteGDBServer::GetFileExists` does the same currently.

Reply:
I'll change just this instance to meet your suggestion